esbuild.js

```diff
@@ -1,10 +1,11 @@
-#!/usr/bin/env -S deno run --unstable --allow-read --allow-write --allow-run --allow-env --allow-net
+#!/usr/bin/env -S deno run --unstable --allow-read --allow-write --allow-run --allow-env --allow-net --allow-ffi
```

Comment:
this is one thing i dislike about `--unstable` -- if/as things move into stable (not always obvi to know what & when) -- it's too easy to leave it in code because you forget why you had to add it in the first place.

i'm trying to write comments where i'm using it more recently -- and trying to slowly pull them out whenever possible.

so.... great eyes!  looks like i was able to just get away w/ a build w/o the `--unstable` flag -- so i'll pull it, thanks! :)